CONTRIBUTING.md

```diff
@@ -130,7 +137,7 @@ pip install -e plugins/sqlfluff-templater-dbt/.
 
 ### Testing
 
-To test locally, SQLFluff uses `tox`. The test suite can be run via:
+To test locally, SQLFluff uses `tox` (check the [requirements](#requirements)!). The test suite can be run via:
```

Comment:
Same feedback here. I know there’s a chance people can miss the early point but personally still find it a little overkill.

```suggestion
To test locally, SQLFluff uses `tox`. The test suite can be run via:
```